### PR TITLE
Character count guideline

### PR DESCRIPTION
**Pull request by crumb1e**
Add a character count guideline to browser title and meta description.

### CODE REVIEW

**Review by tristanward — APPROVED** [review]
Great package

File: src/Webpage/Nova/MetaAttributes.php [hunk 1]
```diff
@@ -17,9 +19,14 @@ protected static function fields()
     {
         return [
             Text::make('H1')->hideFromIndex(),
-            Text::make('Browser Title')->hideFromIndex(),
+            TextCounted::make('Browser Title')
+                ->hideFromIndex()
+                ->maxChars(60)
```
Comment (m2de) [inline src/Webpage/Nova/MetaAttributes.php:24]: Can we remove the hard limit please?

File: src/Webpage/Nova/MetaAttributes.php [hunk 2]
```diff
@@ -17,9 +19,14 @@ protected static function fields()
...
+                ->warningAt(50),
             Text::make('Nav Title')->hideFromIndex(),
-            Textarea::make('Meta Description'),
+            TextareaCounted::make('Meta Description')
+                ->maxChars(300)
```
Comment (m2de) [inline src/Webpage/Nova/MetaAttributes.php:28]: Again, please remove hard limit

**Review by m2de — CHANGES_REQUESTED** [review]
Nice one. Good addition. See small change requests.

**Review by m2de — APPROVED** [review]

**Pull request merged**